packages/jest-worker/src/workers/safeMessageTransferring.ts

```diff
@@ -19,7 +19,15 @@ type TransferringContainer = {
 export function packMessage(message: unknown): TransferringContainer {
   return {
     __STRUCTURED_CLONE_SERIALIZED__: true,
-    data: serialize(message),
+    /**
+     * Use the `json: true` option to avoid errors
+     * caused by `function` or `symbol` types.
+     * It's not ideal to lose `function` and `symbol` types,
+     * but reliability is more important. Additionally,
+     * information about issues with these types will be available
+     * in the error message.
```

Comment:
what is the error message? can we have some test for this case (if nothing else so we know if the case is somehow supported?)

Reply:
Thank you so much for catching that. I'm too immersed in the reporters context and used the incorrect place for saving this information.
Fix https://github.com/jestjs/jest/pull/15191/commits/f4bff33c9f8787a7be55dae064cd4d11790a7b77